tests/unit/workers/tasks/test_import_image.py

```diff
@@ -0,0 +1,22 @@
+from PIL import Image
+from robotoff.utils import get_image_from_url, get_logger, http_session
+
+image = get_image_from_url('https://world.openfoodfacts.org/images/products/802/509/314/0251/4.jpg', error_raise=False)
```

Comment:
good idea, but you should put it in the repository instead (in a tests/data directory for example), we wont to avoid tests depending on an external service.

Reply:
An image with a nutriscore would be more cool ;-) 
Like https://world.openfoodfacts.org/images/products/330/274/003/0949/front_fr.110.400.jpg

(There are plenty, see eg https://world.openfoodfacts.org/label/nutriscore-grade-c)